Assets/Scripts/Weapons/BulletTypes/Bullet.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface Bullet
+{
+    int clipSize { get; }
+    int fireRate { get; } // shots per second
```

Comment:
Why not a float for both fireRate and damage? If we do percentage based power ups than they can be more exact. With integers we have to apply the power up and round.

Reply:
done https://github.com/MaXeraph/Adrenarun/pull/4/commits/9b9ca88baa8cc4ae7fb663f2ec771a73d9f76520